assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -161,6 +161,11 @@ che.docker.always_pull_image=true
 # to be able to launch their own runtimes which are embedded Docker containers.
 che.docker.privileged=false
 
+# This parameter allows to specify custom security options for the created docker container
+# seccomp:unconfined is the default for kubernetes, but not for docker. This is needed
+# for debugging with gdbserver
+che.docker.securityopt=seccomp:unconfined
```

Comment:
Have you tried it on systems that do not support seccomp? Is it just ignored?

Reply:
@garagatyi didn't try it yet, any specific system I should test with?

Reply:
I don't know where to test it and whether it is worth checking, just thought how it can influence users.
I tried to put an invalid value of seccomp profile and another security option. Both failed. So I assume that this would prevent Che starting workspaces if users host doesn't support seccomp.

Reply:
thanks for the feedback, i will test with different values for securityopt and see how it behaves. If unsupported values fail, i will leave it empty as default